src/components/organisms/CreateProjectModal/CreateProjectModal.tsx

```diff
@@ -104,6 +105,7 @@ const CreateProjectModal: React.FC = () => {
       setFormStep(FormSteps.STEP_TWO);
     }
     if (!uiState.fromTemplate && values.rootFolder && values.name) {
+      trackEvent('CREATE_EMPTY_PROJECT');
```

Comment:
event ids should be in a eventconstants.ts file - in case we need to track the same event from multiple places

Reply:
Yeah - for sure I will create a separate PR to handle this.